### PR TITLE
GUVNOR-2452: adding new dependencies needed by new exec server ui

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

File: drools-wb-webapp/pom.xml [hunk 1]
```diff
@@ -907,6 +917,10 @@
             <compileSourcesArtifact>org.guvnor:guvnor-organizationalunit-manager</compileSourcesArtifact>
 
             <!-- Common dependencies -->
+
+            <compileSourcesArtifact>org.kie:kie-api</compileSourcesArtifact>
```
Comment (manstis) [inline drools-wb-webapp/pom.xml]: Do you not also need to ensure/add (some of) these to drools-wb's ```gwt.xml``` file?

**Comment by manstis** [comment]
+1 to merge (when green)

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by csadilek** [comment]
6.4.x: d234dd0290a0853ba1642cca3edd62f0a7afd077

**Pull request closed without merging**